libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/MainMenu.tsx

```diff
@@ -31,7 +31,6 @@ export interface IMainMenuProps {
 
 const settingsIcon: IIconProps = { iconName: "Settings" };
 const infoIcon: IIconProps = { iconName: "Info" };
-const fullscreenIcon: IIconProps = { iconName: "ScaleVolume" };
```

Comment:
why are we removing this?

Reply:
It's not part of the specs. I'm not removing it from error-analysis, only from MAD. @minthigpen FYI

Reply:
the MAD experience won't have a widget version, users will be directed to open the dashboard up in a new tab as there's no way to fit the whole experience in the confines of widget form and by doing so, I think it would degrade the user experience if we tried to force it.